messaging.rst

```diff
@@ -263,13 +263,13 @@ Mediated Transfer
 
 A :term:`Mediated Transfer` is a hash-time-locked transfer. Currently raiden supports only one type of lock. The lock has an amount that is being transferred, a :term:`secrethash` used to verify the secret that unlocks it, and a :term:`lock expiration` to determine its validity.
 
-Mediated transfers have an :term:`initiator` and a :term:`target` and a number of hops in between. The number of hops can also be zero as these transfers can also be sent to a direct partner. Assuming ``N`` number of hops a mediated transfer will require ``6N + 8`` messages to complete. These are:
+Mediated transfers have an :term:`initiator` and a :term:`target` and a number of hops in between. The number of hops can also be zero as these transfers can also be sent to a direct partner. Assuming ``N`` number of hops a mediated transfer will require ``6N + 10`` messages to complete. These are:
 
 - ``N + 1`` mediated or refund messages
 - ``1`` secret request
-- ``N + 1`` secret reveal
+- ``N + 2`` secret reveal
 - ``N + 1`` secret
```

Comment:
Can you rename `secret` to `unlock` as this is the name we use in the spec?

Reply:
Gladly.

Reply:
Done.